### DISCUSSION TITLE
associating orders to vendors

### DISCUSSION THREAD

**Discussion by matsolotl** [swellstores/community#580] [Help] [answered]
I'm setting up a multi-vendor marketplace and am currently stuck at this step:

https://developers.swell.is/guides/build-a-marketplace/extend-swell-models#associating-order-to-vendors

I cannot find any way to do anything with the items array level of the Orders model. There is no way I know of to do with with the "add field" button in the model, and the code I have run through the API consistently returns "Error updating the items array: Resource not found /models/orders/fields/items/fields"

Here is an example of what I've tried running to set up the product.vendor_id formula.

![unnamed](https://github.com/user-attachments/assets/e3af34c3-659d-4bee-9665-4793d2ca6951)

Any help you can provide would be greatly appreciated.

Fwiw, I have had success with all preceding steps in this walkthrough, so any questions you might have about whether I have completed those steps are all "yes" and they are working properly as far as I can tell. I've successfully run scripts to create new products and fetched vendor products lists, etc.

Thanks so much

**Comment by ericingram** [comment, marked as answer]
Try "/:models" instead of "/models"

**Reply by matsolotl** [reply]
Thank you so much!
